### ISSUE TITLE
Code Changes

### ISSUE THREAD

**Issue by smaranjitghose** [smaranjitghose/awesome-portfolio-websites#353]
- [ ] Convert all materialize related code to Bootstrap
- [ ] Keep only the latest version of Bootstrap. For certain pages, there are two versions.
- [ ] Convert entire CSS to Tailwind [ To be done later]
- [ ] Convert entire website to NextJS + React [To be done later]

**Comment by Siddhant-K-code** [comment]
Just a Question, Why we wanted to convert Materialize Codes to BootStrap?

**Comment by satyampandeygit** [comment]
I want to work on this

**Comment by smaranjitghose** [comment]
> Just a Question, Why we wanted to convert Materialize Codes to BootStrap?

We want to have a single CSS framework for the entire project

**Comment by satyampandeygit** [comment]
So we need to convert materialize.min.css to bootstrap.min.css ?

**Comment by smaranjitghose** [comment]
> So we need to convert materialize.min.css to bootstrap.min.css ?

Ya.
But make sure you do it for the related components in the remainder of the pages too.

**Comment by Siddhant-K-code** [comment]
>  But make sure you do it for the related components in the remainder of the pages too.

yes, i tried it there are some components, which we have to change. I will try to do this ASAP

**Comment by smaranjitghose** [comment]
> > But make sure you do it for the related components in the remainder of the pages too.
> 
> yes, i tried it there are some components, which we have to change. I will try to do this ASAP

You are in which program? DWOC or SWoC

**Comment by Siddhant-K-code** [comment]
I am in both program, but, i will do this as SWoC atendee

**Comment by smaranjitghose** [comment]
> I am in both program, but, i will do this as SWoC Participant

No worries. In that case, I can give you labels for both

**Comment by satyampandeygit** [comment]
@Siddhant-K-code how are you doing the same can you share the documents or relevant site for this.

**Comment by Siddhant-K-code** [comment]
> @Siddhant-K-code how are you doing the same can you share the documents or relevant site for this.

Bruteforce method -> comparisons of Official Documentations

**Comment by satyampandeygit** [comment]
> > @Siddhant-K-code how are you doing the same can you share the documents or relevant site for this.
> 
> Bruteforce method -> comparisons of Official Documentations

Thanks!
